### DISCUSSION TITLE
Offline files.... Hard to understand

### DISCUSSION THREAD

**Discussion by pdeioris** [iterate-ch/cyberduck#15129] [Q&A] [answered]
Hello
The logic behind Mountain Duck it's hard for me.
I cannot understand how to manage offline files when I've NO internet connection and volume is not mounted
So, Where are my files when I have NO internet conncetion
I don't meaning cache that are so small size
I mean to have 200 GB offline size...
Where are for offline use?

Thank you

**Comment by tatze96** [comment, marked as answer]
Files can only be managed from within the mounted network drive. The bookmarks can also be mounted without an active network or internet connection within the _Smart Synchronization_ mode.

If a bookmark is mounted offline, it will only display the items previously indexed and those downloaded into the cache either automatically by Mountain Duck after they got opened or manually with the help of the _Keep offline on Local Disk_ option from the context menu. Only the Items that are fully downloaded into the synchronization cache can be opened and edited while using the mount offline.

New files and file changes will be uploaded to the server as soon as an active connection can be established again.

For more information regarding the possible file states, refer to our [wiki page](https://docs.cyberduck.io/mountainduck/connect/sync/#status-of-files).
